unit13-20/unit13.py

```diff
@@ -0,0 +1,13 @@
+'''온라인 할인 쿠폰 시스템 만들기 
+표준 입력으로 가격(정수)과 쿠폰 이름이 각 줄에 입력됩니다.
+Cash3000 쿠폰은 3,000원, Cash5000 쿠폰은 5,000원을 할인합니다.
+쿠폰에 따라 할인된 가격을 출력하는 프로그램을 만드세요.
+(input에서 안내 문자열은 출력하지 않아야 합니다.)
+'''
+money = int(input()) 
+coupon = input()
+if coupon == 'Cash3000':
+    money -= 3000
+if coupon == 'Cash5000':
+    money -= 5000
```

Comment:
```suggestion
if coupon == 'Cash3000':
    money -= 3000
elif coupon == 'Cash5000':
    money -= 5000
```
if문이 두 개일 경우, 모든 if문의 조건을 확인합니다. 위에서 if문을 통해 변화를 주었더라도, 아래에서 또 if문을 돌기 때문에 다음과 같은 상황일 때는 if문을 두 번 쓰는 것보다 if와 elif를 쓰는 것이 더 좋습니다😊